python/pyspark/sql/session.py

```diff
@@ -950,13 +950,13 @@ def dataSource(self) -> "DataSourceRegistration":
 
     @property
     def profile(self) -> Profile:
-        """Returns a :class:`Profile` for performance/memory profiling.
```

Comment:
Otherwise not clickable in Sphinx as: https://spark.apache.org/docs/latest/api/python/reference/pyspark.sql/api/pyspark.sql.SparkSession.profile.html